### ISSUE TITLE
Known issues with finetuned models

### ISSUE THREAD

**Issue by constantinpape** [computational-cell-analytics/micro-sam#484]
This is a growing list of known issues with our finetuned models (referring to the not yet released versions of the models)

- LM Generalists:
    - Creates too large masks in interactive segmentation for some settings, e.g. organoids. See also #473 
        - This is probably due to the training data (TissueNet?). Should update this in the next iteration (e.g. just use suitable subsets of TissueNet that don't have nucleus->cell segmentation) and also train on organoids.

**Comment by constantinpape** [comment]
This should be resolved by the LM generalist v3
